CHANGELOG.md

```diff
@@ -5,6 +5,20 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+
+## [8.0.0]
+### Added
+- Add `wallet_watchAsset` for ERC1155 ([#272](https://github.com/MetaMask/test-dapp/pull/272))
+
+### Changed
+- **BREAKING**: Minimum Node.js version is now 18 ([#264](https://github.com/MetaMask/test-dapp/pull/264))
+- Update to webpack 5 ([#264](https://github.com/MetaMask/test-dapp/pull/264))
+- Update `eth_signTypedData_v4` example ([#274](https://github.com/MetaMask/test-dapp/pull/274))
+- Add `transferFrom` and `approve` to input address for ERC20 contract ([#271](https://github.com/MetaMask/test-dapp/pull/271))
+
+### Fixed
+- Fix signing ([#276](https://github.com/MetaMask/test-dapp/pull/276))
```

Comment:
I think we can omit this one; it wasn't broken in v7.3.1, this fixes something broken in the set of changes since then.

```suggestion
```